ArkEcosystem/laravel-foundation
# refactor: dark mode support for notification overview page

## Summary
https://app.clickup.com/t/1za4ywv

## how to test
Add to MSQ, run dev playbook, login with `hello@basecode.sh`and go to `/user/notifications`

Please note that the notifications icon colors are taken care of in a different PR: https://github.com/ArkEcosystem/laravel-foundation/pull/208

## Checklist
